files/en-us/mdn/structures/page_types/api_event_subpage_template/index.html

```diff
@@ -116,7 +115,7 @@ <h2 id="Specifications">Specifications</h2>
  </tbody>
 </table>
 
-<h2 id="Browser_compatibility_2">Browser compatibility</h2>
+<h2 id="Browser_compatibility">Browser compatibility</h2>
```

Comment:
This ID already exists on 42, which is why it has the number
```suggestion
<h2 id="Browser_compatibility_2">Browser compatibility</h2>
```

Reply:
If a contributor cuts and pasts this page, they would need to change this ID. To eliminate that requirement, I've changed the ID of the heading in line 42 instead.